src/script.js

```diff
@@ -46,10 +69,98 @@ class Item {
   }
 }
 
+// Iterate through the array of tasks and display them in the list
+tasks.forEach((task) => {
+  // eslint-disable-next-line no-new
+  new Item(task.description, task.index);
+});
+
 addButton.addEventListener('click', () => {
   if (inputValue.value !== '') {
+    const newTask = {
+      description: inputValue.value,
+      completed: false,
+      index: tasks.length + 1,
+    };
+    tasks.push(newTask);
     // eslint-disable-next-line no-new
-    new Item(inputValue.value);
+    new Item(newTask.description, newTask.index);
     inputValue.value = '';
   }
 });
+
+// const addButton = document.querySelector('.addButton');
+// const inputValue = document.querySelector('.input');
+// const container = document.querySelector('.container');
+// const itemList = [];
+
+// class Item {
+//   constructor(itemName) {
+//     this.itemName = itemName;
+//     this.createDiv(itemName);
+//     itemList.push(this.itemName);
+//     this.saveItems();
+//   }
+
+//   createDiv(itemName) {
+//     const input = document.createElement('input');
+//     input.value = itemName;
+//     input.disabled = true;
+//     input.classList.add('item_input');
+//     input.type = 'text';
```

Comment:
- [ ] I saw your comment regarding the commented-out code, however, please remove any commented-out code, if this is for the next milestone it can be saved somewhere else temporarily, it will make your code look a lot more organised and easier to read.